base/tests.py

```diff
@@ -219,7 +219,7 @@ class TestGeo(TestCase):
                                       'southwest': {'lat': -0.7461185, 'lng': 8.753189299999999}},
                            'viewport': {'northeast': {'lat': -0.7244695, 'lng': 8.767470699999999},
                                         'southwest': {'lat': -0.7461185, 'lng': 8.753189299999999}},
-                           'location': {'lat': -0.7345885999999999, 'lng': 8.760486199999999},
```

Comment:
Hm, credo che stiamo perdendo di vista lo scopo di questi test. Non e' il nostro compito verificare che Via Toscana abbia sempre le stesse coordinate -- credo sia ragionevole assumere che quello sia il compito di Google. Noi dobbiamo verificare che una chiamata alle API, con un indirizzo valido, ritorni delle coordinate valide, nel formato che ci aspettiamo (ovvero nello schema JSON che ci aspettiamo e siamo in grado di interpretare senza errori). Davvero, non vogliamo interrompere il lavoro di altri perche' ad un certo punto le coordinate di Roma vengono approssimate in modo leggermente diverso ed e' necessario fare il debug di questi test.